adc/streaming.py

```diff
@@ -153,7 +159,7 @@ def __init__(self, broker_url, mode='r', start_at='latest',
 
         if 'w' in mode:
             if len(self.topics) > 1:
-                raise ValueError(f"an AlertBroker in write mode can only have "
+                raise ValueError("an AlertBroker in write mode can only have "
```

Comment:
This fixes a new linter warning in `flake8 v3.8.0`. Unrelated to the main change.